docs/start/integration.md

```diff
@@ -406,44 +414,44 @@ The simplest way to decode a serialized raw transaction is using the [`Transacti
 
 ### Types
 
-As mentioned above Spacemesh implements native account abstraction: all accounts are represented by precompiled smart contracts known as "templates." There are no EOAs (keypair-controlled accounts).
+As mentioned above, Spacemesh implements native account abstraction: all accounts are represented by precompiled smart contracts known as "templates." There are no EOAs (i.e., accounts controlled by key pairs).
 
-Spacemesh currently supports two types of transactions, corresponding to the following two templates:
+Spacemesh currently supports two types of transactions corresponding to the following two templates:
 
-- SingleSig wallet: emulates an EOA. Supports two methods, `Spawn` and `Spend`.
-- MultiSig wallet: same as SingleSig wallet but implements m-of-n key verification. Supports the same two methods, `Spawn` and `Spend`.
+- Single-sig wallet: emulates an EOA and supports two methods, `Spawn` and `Spend`.
+- Multisig wallet: same as Single-sig wallet but implements `m-of-n` key verification. Supports the same two methods, `Spawn` and `Spend`.
 
-Once a transaction has been decoded as above its type may be parsed by looking at the `template` and `method` fields. Here's a legend for mainnet transactions:
+Once a transaction has been decoded as above, its type may be parsed by looking at the `template` and `method` fields. Here is a legend for mainnet transactions:
 
 - SingleSig spawn: template `sm1qqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqg56ypy7`, method `0` (or unset)
 - SingleSig spend: template `sm1qqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqg56ypy7`, method `16`
 - MultiSig spawn: template `sm1qqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqsl0g40s`, method `0` (or unset)
 - MultiSig spend: template `sm1qqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqsl0g40s`, method `16`
 
-Spend transactions have a destination (recipient) account and an amount. These are encoded inside the transaction's raw payload. These data are not currently exposed via the API; we're [working on it](https://github.com/spacemeshos/api/issues/288). For now, in order to fully parse a transaction, you'll have to manually write code such as this [VM code](https://github.com/spacemeshos/go-spacemesh/blob/3d8eedfd44c6f9f8987e604b768596f6001fe703/genvm/templates/wallet/handler.go#L57-L72) that handles tx parsing.
+Spend transactions have a destination (recipient) account and an amount. These are encoded inside the transaction's raw payload. This data is not currently exposed via the API but we're [working on it](https://github.com/spacemeshos/api/issues/288). For now, in order to fully parse a transaction, you will have to manually write code such as this [VM code](https://github.com/spacemeshos/go-spacemesh/blob/3d8eedfd44c6f9f8987e604b768596f6001fe703/genvm/templates/wallet/handler.go#L57-L72) that handles transaction parsing.
```

Comment:
"data" is plural not singular :) https://www.britannica.com/dictionary/eb/qa/Is-Data-Singular-or-Plural-

Reply:
Thanks for the reference, it was definitely informative!

Hmm, it appears to be both as per the link, and I had changed it to its noncount form. But I will accept the change if you feel the plural form better suits the context.

Reply:
you can use either, I'm being pedantic.